tests/tabular/checks/integrity/feature_feature_correlation_test.py

```diff
@@ -78,9 +78,7 @@ def test_feature_feature_correlation_pass_condition(adult_no_split):
 def test_feature_feature_correlation_fail_condition(adult_no_split):
     threshold = 0.5
     num_pairs = 3
-    high_pairs = [('age', 'marital-status'), ('education-num', 'occupation'), ('edu'
-                                                                               ''
-                                                                               'cation', 'education-num'),
+    high_pairs = [('age', 'marital-status'), ('education-num', 'occupation'), ('education', 'education-num'),
```

Comment:
Should add tests for the nan case

Reply:
@noamzbr There's already a test for that

Reply:
where?

Reply:
@JKL98ISR the test is called test_feature_feature_correlation_with_mixed_data

Reply:
so how did nothing change?

Reply:
@JKL98ISR because it was not a bug to begin with, just a decision which of the options we should go for- both return a legit answer. I don't think we should check the actual values.